benchmark/storage/data_table_benchmark.cpp

```diff
@@ -85,8 +94,93 @@ BENCHMARK_DEFINE_F(DataTableBenchmark, ConcurrentInsert)(benchmark::State &state
   state.SetItemsProcessed(state.iterations() * num_inserts_);
 }
 
+// Read the num_reads_ of tuples in a sequential order from a DataTable in a single thread
+// NOLINTNEXTLINE
+BENCHMARK_DEFINE_F(DataTableBenchmark, SequentialRead)(benchmark::State &state) {
+  storage::DataTable read_table(&block_store_, layout_, layout_version_t(0));
+  // Populate read_table by inserting tuples
+  // We can use dummy timestamps here since we're not invoking concurrency control
+  transaction::TransactionContext txn(timestamp_t(0), timestamp_t(0), &buffer_pool_);
+  std::vector<storage::TupleSlot> read_order;
+  for (uint32_t i = 0; i < num_reads_; ++i) {
+    read_order.emplace_back(read_table.Insert(&txn, *redo_));
+  }
+  // NOLINTNEXTLINE
+  for (auto _ : state) {
+    // We can use dummy timestamps here since we're not invoking concurrency control
+    transaction::TransactionContext txn(timestamp_t(0), timestamp_t(0), &buffer_pool_);
```

Comment:
Redundant?

Reply:
If we want everything to be in the same transaction, yes, it's redundant. I'll fix it.